yarn.lock

```diff
@@ -0,0 +1,3675 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Ditto for this one @dannypcheng, for now.  Only saying that because this library is using the lock file provided by NPM instead of using `yarn` as the core package manager.